### DISCUSSION TITLE
What is it Titanium proxy cant do that Telerik FiddlerCore can?

### DISCUSSION THREAD

**Discussion by maxima120** [justcoding121/Titanium-Web-Proxy#830] [General]
I am looking for a way to view ssl traffic from my app inside the app.. I saw Telerik is selling their C# fiddler for $3000 a year.. I thought wow.. Now I found Titanium and it looks like it does same things in essence.. Is it just Telerik are crazy people or am I missing some sort of big picture here?

**Comment by justcoding121** [comment]
For embedding an HTTPS-inspecting proxy in your own app, Titanium covers the same core job: local proxy, decrypt TLS with a root cert, `BeforeRequest`/`BeforeResponse`, modify bodies, etc.

Where FiddlerCore tends to differ (and why the price tag exists):
- Commercial support / SLAs
- More polished helpers, AutoResponder-style tooling, and the Fiddler ecosystem/UI
- Some enterprise packaging/docs

You're not missing a huge "Titanium can't decrypt SSL" gap ? for in-app traffic viewing it usually works. Tradeoff is community/open-source vs paid product support.

If your goal is just see/modify HTTPS inside the app, start with Titanium. Closing this one.
